extensions/scheduler/deployment/src/main/resources/dev-ui/scheduler/qwc-scheduler-log.js

```diff
@@ -7,8 +7,8 @@ import { LogController } from 'log-controller';
  */
 export class QwcSchedulerLog extends LitElement {
     
-    jsonRpc = new JsonRpc("Scheduler");
-    logControl = new LogController(this, "qwc-scheduler-log");
+    jsonRpc = new JsonRpc("Scheduler", false);
```

Comment:
I wonder if it would make sense to also provide a finer grained option? 

Also the logging is very useful during dev UI development/for debugging. So it would be nice to have an easy way to enable the logging for an extension. But I have no idea how this could work :thinking: